packages/ui/src/common/components/page/Sidebar/NavigationLink.tsx

```diff
@@ -119,7 +119,21 @@ const NavigationItemLinkChildren = styled.div`
   ${Overflow.FullDots};
 
   svg {
-    width: 20px;
+    width: 16px;
+    height: 16px;
+
+    path {
+      fill: ${Colors.Black[400]}!important;
```

Comment:
@oleksanderkorn does [`currentcolor`](https://css-tricks.com/currentcolor/) so the hover could be removed ?
```suggestion
      fill: currentcolor !important;
```